db/controller/topic-controller.js

```diff
@@ -0,0 +1,17 @@
+const {
+    selectTopics
+} = require('../model/topics-model');
+
+
+const getTopics = (request, response, next) => {
+    const {  } = request.query
+
+    selectTopics()
+    .then((topics) => {
+        response.status(200).send({topics});
+    })
+    
```

Comment:
Very nice! You're missing a catch block on this one, without which it won't be able to send the error back to app.js. For this endpoint it's not much of an issue because there isn't much that can go wrong, but I noticed you had a Promise.reject in the model